recipes/sdl_mixer/cmake/test_package/test_package.c

```diff
@@ -0,0 +1,42 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <SDL.h>
+#include <SDL_mixer.h>
```

Comment:
I think it would be better to include SDL2 in the path to ensure we look for SDL2 (and not getting SDL by mistake)
```suggestion
#include <SDL2/SDL.h>
#include <SDL2/SDL_mixer.h>
```